docs/RELEASE-NOTES.rst

```diff
@@ -4,6 +4,11 @@ Release Notes for BIG-IP Controller for Kubernetes
 next-release
 ------------
 
+Bug Fixes
+`````````
+* :issues:`735` - Deleted rules from routes and ingresses on the same service not cleaned up properly
+* :issues:`753` - Issue 735 causing the controller to periodically delete and recreate ltm policy objects
```

Comment:
Let's move bug fixes below added functionality to keep the formatting consistent. Also, is 753 related to the url-rewrite issue? If so, let's not add that to the release notes in this PR, we can do that for the specific bug fix MR.

Reply:
Fixed